compat/themify-rules-configuration.json

```diff
@@ -0,0 +1,33 @@
+{
+  "rules": [
+    { "class": "PassThroughRule", "selector": "div" },
```

Comment:
If the video has the `Caption` property specified I still get a warning. I used the following rule to get rid of it:
```javascript
{ "class": "IgnoreRule", "selector": "div.video-caption" }
```

Reply:
I will add a PR to the SDK repo to support captions for Interactive elements since the IA format supports it.